server/src/main/java/org/apache/uniffle/server/LocalStorageChecker.java

```diff
@@ -113,8 +113,9 @@ public boolean checkIsHealthy() {
 
                 totalSpace.addAndGet(total);
                 wholeDiskUsedSpace.addAndGet(total - free);
```

Comment:
Ditto. free -> availableBytes

Reply:
There are many problems with variable naming, I think we can create an issue to track it.

Reply:
I'm OK with this if it involves too many changes.